voxel_grid/src/voxel_grid.cpp

```diff
@@ -46,7 +45,7 @@ namespace voxel_grid {
     size_z_ = size_z; 
 
     if(size_z_ > 16){
-      ROS_INFO("Error, this implementation can only support up to 16 z values (%d)", size_z_); 
+      printf("Error, this implementation can only support up to 16 z values (%d)", size_z_); 
```

Comment:
There are now logging macros in ROS 2, see:

http://docs.ros2.org/beta2/api/rcutils/logging__macros_8h.html

I'm not sure these will be the "final" logging interface that is recommended to be used at this point, but it should be ok to use for now. It's better than `printf` at any rate.